pip-requirements.txt

```diff
@@ -46,7 +46,7 @@ xpartition>=0.2.1
 xarray >= 0.19.0
 
 # developer tools
-conda-lock
+conda-lock<1.0.0
```

Comment:
Requires that we use a version of conda-lock that uses the -p flags to make platform-specific lock files. 

Alternatively, we could switch to using a versions >=1.0, in which we'd need to use a different conda-lock command (render) to implement the approach we've used. See [docs](https://github.com/conda/conda-lock#pre-10-compatible-usage-explicit-per-platform-locks).